templates/static-files.conf

```diff
@@ -0,0 +1,60 @@
+server {
+        listen 80;
+        server_name {{ site_url }};
+
+        {% if use_ssl %}
+        return 301 https://{{ site_url }}$request_uri;
+        {% else %}
+        root {{ site_path }};
+
+        location / {
+                index index.html;
+        }
+
+        include /etc/nginx/locations.d/{{ site_url }}/*.conf;
```

Comment:
This is unsafe for non-US sitenames.  Not a show-stopper but something to be aware of.  It needs to be in the README before this is merged.